spigot_v1_15_R2/src/main/java/com/sk89q/worldedit/bukkit/adapter/impl/Spigot_v1_15_R2.java

```diff
@@ -219,7 +219,7 @@ public Spigot_v1_15_R2() throws NoSuchFieldException {
 
         try {
             Class.forName("org.spigotmc.SpigotConfig");
-            SpigotConfig.config.set("world-settings.worldeditregentempworld.verbose", false);
+            SpigotConfig.config.set("world-settings.faweregentempworld.verbose", false);
```

Comment:
These fields should be adjusted on the Fawe repo as well

Reply:
https://github.com/IntellectualSites/FastAsyncWorldEdit/commit/d6695d23f055ad24b286f7669dffd9b30032ed1d